modules/update-upgrading-cli.adoc

```diff
@@ -129,6 +129,12 @@ date, where the newest update is first in the list. Updates in the history have
 state `Completed` if the rollout completed and `Partial` if the update failed
 or did not complete.
 +
+You can use the `watch` command to follow the status:
++
+----
+$ watch -n 5 './oc get clusterversion -o json|jq ".items[0].status.history"
```

Comment:
Missing single quote in the end
-> `watch -n 5 './oc get clusterversion -o json|jq ".items[0].status.history"'`

Reply:
@gpei Updated. Thank you!